### PR TITLE
Updating Javascript article to Spin 2.0

### PR DESCRIPTION
**Pull request by mikkelhegn**
✅ content/spin/v2/javascript-components.md

---

Screenshot 2023-11-02 at 08 23 21

---

Screenshot 2023-11-02 at 08 24 08

### CODE REVIEW

File: content/spin/v2/javascript-components.md [hunk 1]
```diff
@@ -188,18 +189,23 @@ export const handleRequest: HandleRequest = async function (request: HttpRequest
 
     return {
         status: 200,
+<<<<<<< Updated upstream
```
Comment (karthik2804) [inline content/spin/v2/javascript-components.md:192]: The article has a few git diffs

File: content/spin/v2/javascript-components.md [hunk 2]
```diff
@@ -188,18 +189,23 @@ export const handleRequest: HandleRequest = async function (request: HttpRequest
...
         headers: {"foo": "bar"},
         body: encoder.encode("Hello from JS-SDK").buffer
+=======
```
Comment (karthik2804) [inline content/spin/v2/javascript-components.md:195]:
```suggestion
```

File: content/spin/v2/javascript-components.md [hunk 3]
```diff
@@ -188,18 +189,23 @@ export const handleRequest: HandleRequest = async function (request: HttpRequest
...
+        headers: {"content-type": "text/plain"},
+        body: "Hello from TS-SDK"
+>>>>>>> Stashed changes
```
Comment (karthik2804) [inline content/spin/v2/javascript-components.md:198]:
```suggestion
```

File: content/spin/v2/javascript-components.md [hunk 4]
```diff
@@ -248,6 +267,7 @@ domains the component is allowed to make HTTP requests to:
 <!-- @nocpy -->
 
 ```toml
+<<<<<<< Updated upstream
```
Comment (karthik2804) [inline content/spin/v2/javascript-components.md:270]:
```suggestion
```

**Review by karthik2804 — COMMENTED** [review]
Looking good overall but some git conflicts have been checked in.

**Comment by mikkelhegn** [comment]
@karthik2804 Working on it, and closed by accident. Give me a few minutes :-)

**Review by tpmccallum — APPROVED** [review]
LGTM

**Pull request merged**